packages/shared/src/react/hooks/useOrganization.tsx

```diff
@@ -264,7 +269,7 @@ const undefinedPaginatedResource = {
  * }
  * ```
  */
-export const useOrganization: UseOrganization = params => {
+export function useOrganization<T extends UseOrganizationParams>(params?: T): UseOrganizationReturn<T> {
```

Comment:
No behavior change, just removing a type alias and explicitly making it a `function` (sometimes typedoc can be tripped up by using a `const` for a function)